content/300-guides/150-testing/100-unit-testing.mdx

```diff
@@ -109,45 +108,45 @@ Another popular pattern that can be used is dependency injection.
 
 1. Create a `context.ts` file and add the following:
 
-  ```ts file=context.ts
-  import { PrismaClient } from '@prisma/client'
-  import { MockProxy, mockDeep } from 'jest-mock-extended'
+```ts file=context.ts
+import { PrismaClient } from '@prisma/client'
+import { MockProxy, mockDeep } from 'jest-mock-extended'
 
-  export type Context = {
-    prisma: PrismaClient
-  }
+export type Context = {
+  prisma: PrismaClient
+}
 
-  export type MockContext = {
-    prisma: MockProxy<PrismaClient>
-  }
+export type MockContext = {
+  prisma: MockProxy<PrismaClient>
+}
 
-  export const createMockContext = (): MockContext => {
-    return {
-      prisma: mockDeep<PrismaClient>(),
-    }
+export const createMockContext = (): MockContext => {
+  return {
+    prisma: mockDeep<PrismaClient>(),
   }
-  ```
+}
+```
 
 <Tip>
 
-  If you find that you're seeing a circular dependency error highlighted through mocking the client, try adding `"strictNullChecks": true`
-  to your `tsconfig.json`.
+If you find that you're seeing a circular dependency error highlighted through mocking the client, try adding `"strictNullChecks": true`
+to your `tsconfig.json`.
 
 </Tip>
 
 2. To use the context, you would do the following in your test file:
 
-  ```ts
```

Comment:
investigating this…

Reply:
Probably: https://github.com/prisma/docs/pull/1995/files#r665270238